protos/protos/public/modeldb/ExperimentRunService.proto

```diff
@@ -73,6 +73,7 @@ message Observation {
         common.Artifact artifact = 2;
     }
     int64 timestamp = 3;
+    google.protobuf.Value epoch_number = 4;
```

Comment:
```suggestion
    google.protobuf.NumberValue epoch_number = 4;
```
Avoids having to worry about arbitrary values.

Reply:
there isn't a Number Value at https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/struct.proto

Reply:
added check and test

Reply:
Read the documentation wrongly. My bad.